core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/puppeteer-utils.ts

```diff
@@ -183,6 +183,14 @@ export class BaseUser {
       await this.page.click(selector);
     }
   }
+  /**
+   * The function selects all text content and delete it.
+   */
+  async clearAllTextFrom(selector: string): Promise<void> {
+    await this.waitForElementToBeClickable(selector);
+    await this.page.click(selector, {clickCount: 3});
```

Comment:
Add a one-line comment above this line: Clicking three times on a line of text selects all the text.

(Reason: I don't think this is common knowledge.)

Reply:
done